AMaDda/Screens/Onboarding/OnboardingTwoViewController.swift

```diff
@@ -52,6 +53,14 @@ class OnboardingTwoViewController: UIViewController {
         showNotificationLabel.text = "\(notificationCount)일"
     }
     
+    @objc func buttonAction(sender: UIButton!) {
+        let mainVC = MainViewController()
+        navigationController?.pushViewController(mainVC, animated: true)
+        navigationController?.isNavigationBarHidden = true
+        UserDefaults.standard.set(notificationCount, forKey: "notificationCount")
+        UserDefaults.standard.set(true, forKey: "checkedOnboarding")
```

Comment:
p1;
이 두가지 key 값도 위에 말한것처럼 UserDefaults+Extension에 선언해둔다면
```suggestion
        UserDefaults.standard.notificationCount = notificationCount
        UserDefaults.standard.checkedOnboarding = true
```
로 간략화 시킬 수 있습니다!

Reply:
suggestion을 통해서 친절하게 사용방법까지 알려주셔서 감사합니다!!! 🥹

Reply:
[dfbeb43](https://github.com/DeveloperAcademy-POSTECH/MC3-Team14-HyojaDongHyojadeul/pull/63/commits/dfbeb430d1e690b6110873f390a5bd833fa4ca24)